### DISCUSSION TITLE
Kerberos support?

### DISCUSSION THREAD

**Discussion by KongNan** [hvac/hvac#1147] [Q&A] [answered]
Awesome! Does HVAC support the "very hands-on auth method" Kerberos?

https://developer.hashicorp.com/vault/docs/auth/kerberos#prerequisites

**Comment by briantist** [comment]
Hi @KongNan ,

~I have not used it myself but the library does support Kerberos, See:~
- https://hvac.readthedocs.io/en/stable/usage/auth_methods/kubernetes.html#
- https://hvac.readthedocs.io/en/stable/source/hvac_api_auth_methods.html#hvac.api.auth_methods.Kubernetes

**Reply by KongNan** [reply]
Thank you for your reply!  Kerberos != K8s :)

**Reply by briantist** [reply]
wow.. that is very embarrassing, I really should not have replied before having coffee 🤦

**Comment by briantist** [comment, marked as answer]
Right now the library does not have Kerberos support. There is an open request, and an open PR although the PR was not completed and has not seen activity in a while:
- https://github.com/hvac/hvac/issues/698
- https://github.com/hvac/hvac/pull/1027
